include/lsst/meas/extensions/shapeHSM/HsmMomentsControl.h

```diff
@@ -69,9 +69,24 @@ class HsmSourceMomentsRoundControl : public HsmSourceMomentsControl {
 
 class HsmPsfMomentsControl {
 public:
-    HsmPsfMomentsControl() {}
+    LSST_CONTROL_FIELD(debias, bool, "Debias PSF moments?");
```

Comment:
I'm not a fan of the fact that this makes it possible to turn the debiasing off the plugin we've named "debiased".

I think it would be a lot cleaner to split the `HsmPsfMomentsAlgorithm` class into a base + 2 derived triplet, where the new base class (best to keep the name of the not-debiased concrete algorithm the same) implements `measure` by calling a new `getPsfImage` virtual method that each derived class would implement.

Reply:
Sounds good.  I haven't looked in detail yet, but I think the same comment applies to `HsmSourceMomentsRound`?  I.e., I could manipulate its control to set round=false?  If so I'll refactor that one too.

Reply:
Yes, please do.  I had forgotten all about that one.